FileManager.php

```diff
@@ -46,6 +46,16 @@ final class FileManager
      */
     const STRATEGY_RENAME_INC = 16;
 
+    /**
+     * When moving files, on name conflict, by default move is prevented if
+     * the files shares the same content (tested via cheksum).
+     * Using this flag you will prevent this checksum test and allow
+     * a move on the same file. This can be used to ensure the rename()
+     * operation is really removing the source file from the initial
+     * directory, as it may not be the case withour this flag.
+     */
+    const ALLOW_MOVE_ON_SAME_CONTENT = 32;
```

Comment:
There already the MOVE_CONFLICT_OVERWRITE flag, it would be best to reuse it. https://github.com/makinacorpus/filechunk-bundle/blob/master/FileManager.php#L35

Reply:
OK this is not exactly the same, you're avoiding the duplicate check, let's merge this.